templates/bootstrap/index.tmpl

```diff
@@ -13,7 +13,7 @@
     <div class="navbar navbar-default navbar-static-top" role="navigation">
       <div class="container">
         <div class="navbar-header">
-          <a class="navbar-brand">Cucumberjs Report</a>
+          <a class="navbar-brand"></a>
```

Comment:
I believe we should parameterized the brand `title` instead of completely removing it. If parameter `title` is not passed in the `options` then show the default `Cucumberjs Report` otherwise show the customized title (as a future enhancement). Let me know if you agree.